example_avro_ingestion_gcs-cdap-data-pipeline.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
let's put this in an examples directory instead of at the top level,

Reply:
done